lib/miq_expression.rb

```diff
@@ -1399,13 +1403,7 @@ def to_arel(exp, tz)
         ids = tag.model.find_tagged_with(:any => parsed_value, :ns => tag.namespace).pluck(:id)
         tag.model.arel_attribute(:id).in(ids)
       else
-        raise unless field.associations.one?
```

Comment:
Do we still need this?

Reply:
we no longer limit it to relationships that are 1 deep.

Reply:
Does this make this branch a bit of an `enhancement` then?

Reply:
thanks though - I added tests for this

Reply:
also found a bug. (and added support for habtm and belongs_to)

Reply:
I simplified it again (pushed off for another PR)